src/Streams/FileReader.cpp

```diff
@@ -60,8 +60,7 @@ namespace Stream
 	void FileReader::SeekRelative(int64_t offset) 
 	{
 		if (offset < 0) {
-			const int mask = offset >> ( sizeof(int64_t) * CHAR_BIT - 1 );
-			const uint64_t offsetAbsValue = (offset ^ mask) - mask;
+			const uint64_t offsetAbsValue = -offset;
```

Comment:
You removed the bit twiddling...

Reply:
:scream: 

Hey..., what does this "Resolve conversation" button do? I've never noticed it before....